examples/deferred_processing/automated_deferred_processing/README.md

```diff
@@ -0,0 +1,348 @@
+and create environment variables for use by the pipeline.
+If you are unfamiliar with those things,
+it may be helpful to refer to the following documentation as you work through the example.
+* [Pachyderm authentication documentation](https://docs.pachyderm.com/latest/enterprise/auth/)
+* [Kubernetes documentation on
+Secrets](https://kubernetes.io/docs/concepts/configuration/secret/),
+* [pachctl create secret](https://docs.pachyderm.com/latest/reference/pachctl/pachctl_create_secret/) command
+* [Pachyderm documentation on adding a volume to a pipeline](https://docs.pachyderm.com/latest/how-tos/mount-volume/) using [pod_patch](https://docs.pachyderm.com/latest/reference/pipeline_spec/#pod-patch-optional)
+* the [jq utility](https://stedolan.github.io/jq/manual/) for transforming json files in shell scripts
+
+You need to have Pachyderm v1.9.8 or later installed on your computer or cloud platform. 
+See [Deploy Pachyderm](https://docs.pachyderm.com/latest/deploy-manage/deploy/).
+
+It helps to have some understanding of Makefiles, 
+if you plan on using those in your infrastructure.
+Basic Unix shell scripting skills are assumed.
+
+## Pipelines
+
+This example uses the same DAG as in the deferred processing example,
+with the addition of a cron pipeline 
+for periodically moving the `dev` branch to `master`.
+
+For details on the deferred processing example DAG,
+please refer to [that example](../deferred_processing_plus_transactions).
+
+### Branch mover, without authentication
+
+The cron pipeline is called `branch-mover`. 
+It is configured to run, by default, every 1 minute, 
+per its tick input:
+
+```
+  "input": {
+    "cron": {
+        "name": "tick",
+        "spec": "@every 1m",
+        "overwrite": true
+    }
+  },
+```
+
+The code for moving the branch is very simple, 
+if the Pachyderm cluster does not have authentication activated.
+
+Using the official Pachyderm pachctl image, 
+the transform first updates the default `pachctl` config
+so `pachctl` can talk directly to `pachd` in the cluster.
+It uses the `kubedns` name for `pachd`, 
+and the internal Service port of `650`. 
+
+```
+          "echo '{\"pachd_address\": \"grpc://pachd:650\"}' | pachctl config set context default --overwrite",
+```
+
+The next command moves the `master` branch on `edges_dp` to point to `dev`,
+just like in the deferred processing example.
+
+```
+          "pachctl create branch edges_dp@master --head dev"
+```
+
+This is all the cron pipeline needs to do, 
+without authentication enabled. 
+The transform in the pipeline spec `branch-mover-no-auth.pipeline`
+ends up looking like this:
+
+```
+  "transform": {
+      "cmd": ["sh" ],
+      "stdin": [
+          "echo '{\"pachd_address\": \"grpc://pachd:650\"}' | pachctl config set context default --overwrite",
+          "pachctl create branch edges_dp@master --head dev"
+      ],
+    "image": "pachyderm/pachctl:1.11.0"
+  }
+```
+
+### Branch mover, with authentication
+
```

Comment:
Use the instructions in this section, if you have authentication enabled in your Pachyderm cluster.

Reply:
I'm going to end up turning all references to "authentication enabled" to "access controls activated" in this document, so I'm consistent with your first comment...